### ISSUE TITLE
Commputation Model

### ISSUE THREAD

**Issue by fauzan132** [Cerebro409/EEG-Classification-Using-Recurrent-Neural-Network#2]
Hello, my name is Fauzan Ar Rasyid, I am university student and i am working on my final project about classification neuropsychologist with EEG using Recurrent Neural Networks (LSTM) but i have a problem about the accuracy.

When i'm using your model and i train the data, the accuracy stuck under 50% for training data, i have tried using different data for training data, but the result still same, do you have any idea for my problem? i have tried all the possibilities

Fauzan
